manifests/istio/istio/overlays/https-gateway/params.env

```diff
@@ -0,0 +1 @@
+gatewaySelector=ingressgateway
```

Comment:
Since this is not exposed to the user, why do you need to create this file instead of including it in the `kf-istio-resources.yaml` directly?

Reply:
it's just a copy from manifest directory, trying to keep it the same, so it would be easier to merge with new versions of manifests

Reply:
Why do you need to copy it?

Reply:
because I need to add changes to it